src/components/Form/Picklist.js

```diff
@@ -120,6 +123,8 @@ export class PicklistRaw extends Component {
   }
 }
 
+PicklistRaw.size = PICK_LIST_SIZE;
```

Comment:
Why do we need to set this here? There is a lot of indirection going on with the `constants` file and the static property here

Reply:
We need this static property to hide slds internals from the user of our component:
```
<Picklist
...
size={Picklist.size.SMALL}
>
```

Reply:
We do not do this in any other place and this is not necessary as Salesforce does not change these terms and they are also used all over the place in `slds`-classes, which our users (us) have to use all the time. This introduces a new concept that is not documented well and only applied in this component.

Reply:
If you think that this should be changed, please open a ticket and we can discuss this as a team. Going forward and changing this in one place while leaving all other components as is is not ideal